great_expectations/core/expectation_validation_result.py

```diff
@@ -441,7 +441,7 @@ class ExpectationSuiteValidationResultMeta(TypedDict):
     expectation_suite_name: str
     great_expectations_version: str
     run_id: RunIdentifier
-    validation_id: Optional[str]
+    validation_id: str
```

Comment:
@cdkini was this required to get things working? I think we want this to be nullable for when you run `batch.validate(suite)`, right?